x-pack/plugins/actions/server/actions_client.ts

```diff
@@ -196,17 +196,6 @@ export class ActionsClient {
   }: CreateOptions): Promise<ActionResult> {
     const id = options?.id || SavedObjectsUtils.generateId();
 
-    if (this.preconfiguredActions.some((preconfiguredAction) => preconfiguredAction.id === id)) {
```

Comment:
Moved the authorization checks to the top of the function following the same pattern we have in the other functions.